cam:
  width: 320
  height: 240
  left:
    K: [159.99999300617776, 0.0, 160.0, 0.0, 159.99999300617776, 120.0, 0.0, 0.0, 1.0]
  right:
    K: [159.99999300617776, 0.0, 160.0, 0.0, 159.99999300617776, 120.0, 0.0, 0.0, 1.0]

line_det:
  bird_pixel_size: 0.01
  max_line_width: 0.5


